Readme.md

```diff
@@ -254,6 +261,10 @@ test('async test', () => {
   // Your async tests
 })
 ```
+Note that you can also do this per file, outside any `describe` and `test`:
```

Comment:
Will this check that each test case has at least one assertion?

Reply:
Yes it does! And it doesn't conflict with more specific `expect.assertions(3)` declarations either.

Reply:
I'll add a note to clarify this point.